test/integration/mixer_fault_test.cc

```diff
@@ -545,16 +546,12 @@ TEST_F(MixerFaultTest, FailClosedAndSendPolicyResponseSlowly) {
   // Every response was a 5xx class
   EXPECT_EQ(0, client->class2xxResponses());
   EXPECT_EQ(0, client->class4xxResponses());
-  EXPECT_EQ(requests_to_send, client->class5xxResponses());
+  EXPECT_EQ(client->class5xxResponses(), requests_to_send);
```

Comment:
I think EXPECT_EQ(requests_to_send, client->class5xxResponses()) is consistent with other EXPECT_EQ(expected result, tested result).

Reply:
Yeah, I agree with @JimmyCYJ that the style in all those checks is a bit inconsistent.

But taking into account the range tests for macOS, I think that the tests above should move runtime-values to the left-hand side for readability, i.e.:
```
EXPECT_EQ(client->class2xxResponses(), 0);
EXPECT_EQ(client->class3xxResponses(), 0);
```

Reply:
The readability issues I will fix

(they are consistent with yoda conditions but in this case provide no actual value)

Reply:
done